syft/frameworks/torch/he/fv/evaluator.py

```diff
@@ -9,6 +10,30 @@
 from syft.frameworks.torch.he.fv.plaintext import PlainText
 
 
+class ParamTypes(Enum):
+    """Enumeration for type checking of parameters.
+    """
+
+    CTCT = 1
+    PTPT = 2
+    CTPT = 3
+    PTCT = 4
+
+
+def _typecheck(op1, op2):
+    """Check the type of parameters used and return correct enum type."""
+    if isinstance(op1, CipherText) and isinstance(op2, CipherText):
+        return ParamTypes.CTCT
+    elif isinstance(op1, PlainText) and isinstance(op2, PlainText):
+        return ParamTypes.PTPT
+    elif isinstance(op1, CipherText) and isinstance(op2, PlainText):
+        return ParamTypes.CTPT
+    elif isinstance(op1, PlainText) and isinstance(op2, CipherText):
+        return ParamTypes.PTCT
+    else:
+        return None
```

Comment:
@bcebere I have tried to update it as per your review, please check and suggest if anything better can be done.

Reply:
Maybe throw directly, instead of renurning None?

Reply:
But then I would have to remove the name of operation. Can you suggest another error message without operation name.